metainf/pom.xml

```diff
@@ -61,14 +61,6 @@
   </dependencies>
   <build>
     <plugins>
-      <plugin>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
-          <compilerVersion>1.6</compilerVersion>
-          <source>1.6</source>
-          <target>1.6</target>
```

Comment:
Java 11 and higher can't build with Java 6 compat.

Reply:
@elucash is happy to go with java 8+ support :) 

https://github.com/immutables/immutables/pull/1247#issuecomment-732422495